Makefile

```diff
@@ -46,13 +51,8 @@ ifneq ($(HAS_WEBAPP),)
 endif
 
 ifneq ($(HAS_SERVER),)
-	@if ! [ -x "$$(command -v golangci-lint)" ]; then \
-		echo "golangci-lint is not installed. Please see https://github.com/golangci/golangci-lint#install for installation instructions."; \
-		exit 1; \
-	fi; \
-
 	@echo Running golangci-lint
-	golangci-lint run ./...
+	golangci-lint run ./... --verbose
```

Comment:
`--verbose` can make CI take longer to run. Are we sure we want this to run this way in CI?

Reply:
It only takes some seconds to run and it's nice to have the verbosity in place